### PR TITLE
Possible Invasion Rewards

### LINKED ISSUE #70: feat: Grunt rewards
Super excessive nice to have but being able to show the possible grunt rewards. Highly recommend stealing the json format from pmsf, a few projects have already done so, then it's easier for community to update in the future.

8BC64673-B258-4256-AF8F-6975B06E303F

C3DD35CA-5C2A-4181-849F-D53FA9255532

https://github.com/pmsf/PMSF/blob/master/static/data/grunttype.json

### PR DESCRIPTION
**Pull request by versx**
image
Fix #70

### CODE REVIEW

**Comment by clburlison** [comment]
Working over here as well but kinda wonky depending on the icon set. Some icon repos might not have perfect square icons so need to do something.

IMG_6209

**Comment by Mygod** [comment]
@clburlison Oh yeah thanks for reminding me I need to work on this.

File: static/js/index.js [hunk 1]
```diff
@@ -3055,6 +3061,58 @@ function getPokestopPopupContent (pokestop) {
...
+        content += `
+        </div>
+    </div>`;
+	}
+    return content;
+}
+
+function showHideGruntEncounter() {
+    var x = document.getElementsByClassName('grunt-encounter-wrapper')
+    var i
```
Comment (Mygod) [inline static/js/index.js:3106]: Why the inconsistency of `;`'s and `var`/`let`?

Comment (versx) [reply static/js/index.js:3106]: Oops

**Pull request merged**